modules/develop/pages/intro.adoc

```diff
@@ -83,4 +83,12 @@ ifdef::flag-devex-analytics[]
 The Analytics Service is a parallel data management capability for Couchbase Capella, designed to run ad-hoc, analytical queries efficiently over many records.
 
 * xref:analytics:introduction.adoc[Introduction to Analytics]
-endif::flag-devex-analytics[]
\ No newline at end of file
+endif::flag-devex-analytics[]
+
+
+== Integrations
+
+Integrations that Couchbase or partners have developed to enhance Couchbase, Capella, and partner services --
+a vital tool for many larger development projects.
+
+* xref:server:develop:integrations.adoc[]
```

Comment:
See comments on #72 . I'm really not keen on the idea of an xref in a top-level landing page on the Capella side taking users over the server side without much warning. Remember that docs-devex is specifically for information which is shared between cloud and server. If you're including the Integrations page in docs-devex on the server side, and you're linking to it from the intro page on the Capella side, you should include the Integrations page on the Capella side also.

Reply:
(Which would mean that this link would then become

```suggestion
* xref:develop:integrations.adoc[]
```

in _this_ branch also.)